lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -458,10 +475,19 @@ def main():
         module.fail_json(msg="Exception describing service '" + module.params['name'] + "' in cluster '" + module.params['cluster'] + "': " + str(e))
 
     results = dict(changed=False)
+
+    if module.params['launch_type']:
+        if not service_mgr.ecs_api_supports_launch_type():
```

Comment:
Rather than importing LooseVersion in the module you could use `if not module.botocore_at_least('1.8.4'):` (added a few days ago in 7f8654d586be176a355a685c249eb28fd5a17d0d)